src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,41 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  width: $col-quantity * $day-size + $day-gap * 6;
+  padding: 20px $calendar-padding;
+
+  @each $day,$i in $days-of-week {
+    &--start-day_#{$day} &__day:first-child {
+      margin-left: ($day-size + $day-gap) * $i;
+    }
+  }
+
+  @for $day from 28 through 31 {
+    &--month-length_#{$day} &__day:nth-child(n + #{$day + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    @include flexBox;
+    @include dayCell;
+
+    box-sizing: border-box;
```

Comment:
The better solution is to use that property in reset or normalize style file.

Reply:
correct